subo/command/dev.go

```diff
@@ -42,7 +43,7 @@ func DevCmd() *cobra.Command {
 				util.LogInfo("Running Atmo with debug logging")
 			}
 
-			dockerCmd := fmt.Sprintf("docker run -v=%s:/home/atmo -e=ATMO_HTTP_PORT=%s %s -p=%s:%s suborbital/atmo:%s atmo", bctx.Cwd, port, envvar, port, port, bctx.AtmoVersion)
+			dockerCmd := fmt.Sprintf("docker run -v=%s:/home/atmo -e=ATMO_HTTP_PORT=%s %s -p=%s:%s suborbital/atmo:%s atmo", bctx.Cwd, port, envvar, port, port, release.DeltavVersion)
```

Comment:
is there a reason to continue labeling the docker image `atmo` instead of `deltav`

Reply:
Good catch